pkg/model/components/context.go

```diff
@@ -125,8 +126,16 @@ func Image(component string, clusterSpec *kops.ClusterSpec) (string, error) {
 		return "gcr.io/google_containers/kubedns-amd64:1.3", nil
```

Comment:
We do not want to remap this image, as it is already in addons.  Why add another one?